src/helper/class-ee-site.php

```diff
@@ -625,36 +632,28 @@ public function reload( $args, $assoc_args, $whitelisted_containers = [], $reloa
 		\EE\Utils\delem_log( 'site reload start' );
 		$args = auto_site_name( $args, 'site', __FUNCTION__ );
 		$all  = \EE\Utils\get_flag_value( $assoc_args, 'all' );
-		if ( ! array_key_exists( 'nginx', $reload_commands ) ) {
-			$reload_commands['nginx'] = 'nginx sh -c \'nginx -t && service openresty reload\'';
-		}
+		$reload_commands['php'] = "bash -c 'kill -USR2 1'";
+		$reload_commands['nginx'] = "sh -c 'nginx -t && service openresty reload'";
 		$no_service_specified = count( $assoc_args ) === 0;
 
 		$this->site_data = get_site_info( $args );
 
-		chdir( $this->site_data['site_fs_path'] );
-
 		if ( $all || $no_service_specified ) {
-			$this->reload_services( $whitelisted_containers, $reload_commands );
+			foreach ( $whitelisted_containers as $container ) {
+				if ( ! \EE_DOCKER::docker_compose_exec( $this->site_data['site_fs_path'], $container, $reload_commands[ $container ] ) ) {
+					throw new \Exception( 'There was some error in docker-compose exec.' );
```

Comment:
Again something useful can be added here like: `Reloading service for $container failed.`